kotlin/build.gradle

```diff
@@ -15,45 +15,44 @@
  */
 buildscript {
   ext.versions = [
-      'agp': '3.5.0-beta05',
-      'buildTools': '28.0.3',
-      'compileSdk': '28',
-      'lifecycle': '2.0.0',
-      'minSdk': 21,
-      'targetSdk': 28,
-      'sourceCompatibility': JavaVersion.VERSION_1_8,
-      'targetCompatibility': JavaVersion.VERSION_1_8,
-
-      'coordinators': '0.4',
-      'rxandroid2': '2.1.0',
-      'timber': '4.7.1',
-
-      'detektPlugin': '1.0.0-RC16',
+      'targetSdk': 29,
       'dokka': '0.9.18',
-      'hamcrest': '1.3',
-      'intellijAnnotations': '13.0',
-      'junit': '4.12',
       'kotlin': '1.3.41',
       'kotlinCoroutines': '1.3.0-M2',
-      'ktlintPlugin': '5.1.0',
-      'mavenPublishPlugin': '0.8.0',
-      'mockito': '2.7.5',
-      'mockitoKotlin': '1.5.0',
-      'rxjava2Extensions': '0.20.4',
-      'truth': '1.0',
   ]
 
+  rootProject.ext.defaultAndroidConfig = {
+    compileSdkVersion versions.targetSdk
+    buildToolsVersion '29.0.0'
+
+    compileOptions {
+      sourceCompatibility JavaVersion.VERSION_1_8
+      targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    defaultConfig {
+      minSdkVersion 21
+      targetSdkVersion versions.targetSdk
+      versionCode 1
+      versionName "1.0"
+    }
+  }
+
   ext.deps = [
-      'android_gradle_plugin': "com.android.tools.build:gradle:${versions.agp}",
+      'android_gradle_plugin': "com.android.tools.build:gradle:3.5.0-beta05",
+
+      'androidx': [
```

Comment:
Thanks! I had made a mental note to do this the other day and forgot to write it down.